test/library/randomized_benchmarking/test_randomized_benchmarking.py

```diff
@@ -563,9 +563,9 @@ def test_two_qubit(self):
         # average number of CX gate per Clifford is 1.5.
         # Since this is two qubit RB, the dep-parameter is factored by 3/4.
         epc = expdata.analysis_results("EPC")
-        # Allow for 50 percent tolerance since we ignore 1q gate contribution
+        # Allow for 30 percent tolerance since we ignore 1q gate contribution
         epc_expected = 1 - (1 - 3 / 4 * self.p2q) ** 1.5
-        self.assertAlmostEqual(epc.value.n, epc_expected, delta=0.5 * epc_expected)
+        self.assertAlmostEqual(epc.value.n, epc_expected, delta=0.3 * epc_expected)
```

Comment:
How are these tolerances calculated? Do these tests all have similar probabilities of failing?

Reply:
I'm not sure how the original values are calculated. I guess they are just determined empirically. I don't stick to the value `0.3`. If you know a better value, I'll switch to it. Anyway, at least, I believe `two_qubits`, `two_qubit_parallel` and `two_qubit_with_cz` should have the same delta value as 1q gate contribution to EPC in them should be the same (so all should have the similar probabilities of failing).